pages/[word].tsx

```diff
@@ -0,0 +1,16 @@
+import Head from "next/head";
+import { useRouter } from 'next/router';
+
+export default function Word() {
+  const router = useRouter();
+  const { word } = router.query;
+  return (
+    <>
+      <Head>
+        <title>{word} | Xaitsnoo Community Dictionary Project</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
```

Comment:
This is fine for a boilerplate PR but when you make the page shell, I strongly suggest we abstract this in a meaningful way so we dont have "Xaitsnoo Community Dictionary Project" hard-coded in many places.